packages/react-google-maps-api/src/components/dom/OverlayView.tsx

```diff
@@ -126,32 +128,25 @@ export class OverlayView extends React.PureComponent<OverlayViewProps, OverlayVi
 
   onRemove = (): void => {
     this.mapPaneEl = null
```

Comment:
Here is typescript error, at least in TS@4.0.2. I have replaced this line with:
```
this.setState(() => ({
      paneEl: null
    }))
``` 
to publish 1.10.0